docs/guide/relationships/one-to-one-poly.md

```diff
@@ -0,0 +1,98 @@
+model. You may define such a relationship using 2 types of relationship attributes, `MorphOne` and `MorphTo`.
+
+## Defining A One To One Polymorphic Relationship
+
+To define this relationship, for example, a `User` or `Post` model might be associated with one `Image`, we define a
+`morphOne` field to the `User` and `Post` models.
+
+```js
+class Image extends Model {
+  static entity = 'images'
+
+  static fields () {
+    return {
+      id: this.number(0),
+      url: this.string(''),
+      imageableId: this.number(0),
+      imageableType: this.string(''),
+    }
+  }
+}
+
+class User extends Model {
+  static entity = 'users'
+
+  static fields () {
+    return {
+      id: this.number(0),
+      name: this.string(''),
+      image: this.morphOne(Image, 'imageableId', 'imageableType')
+    }
+  }
+}
+
+class Post extends Model {
+  static entity = 'posts'
+
+  static fields () {
+    return {
+      id: this.number(0),
+      title: this.string(''),
+      image: this.morphOne(Image, 'imageableId', 'imageableType')
+    }
+  }
+}
+```
+
+The first argument passed to the `morphOne` method is the name of the model, the second argument is the name of the
+field which will contain the `id` of the model, and the third argument is the name of the field which will contain the
+`entity` of the parent model. The third argument is used to determine the "type" of the related parent model.
+
+Additionally, Vuex ORM assumes that the foreign key should have a value matching the `id`
+(or the custom `static primaryKey`) field of the parent. In other words, Vuex ORM will look for the value of the user's
+`id` field in the `imageableId` field of the `Image` record. If you would like the relationship to use a value other
+than `id`, you may pass a fourth argument to the `morphOne` method specifying your custom key:
+
+```js
+class User extends Model {
+  static entity = 'users'
+
+  static fields () {
+    return {
+      id: this.number(0),
+      userId: this.string(''),
+      name: this.string(''),
+      image: this.morphOne(Image, 'imageableId', 'imageableType', 'userId')
+    }
+  }
+}
+```
+
+## Defining The Inverse Of The Relationship
+
+So, we can access the `Image` model from our `User` or `Post`. Now, let's define a relationship on the `Image` model
+that will let us access the model which owns the image. We can define the inverse of a `morphOne` relationship using the
+`morphTo` attribute:
+
+```js
+class Image extends Model {
+  static entity = 'images'
+
+  static fields () {
+    return {
+      id: this.number(0),
+      url: this.string(''),
+      imageableId: this.number(0),
+      imageableType: this.string(''),
+      imageable: this.morphTo('imageableId', 'imageableType'),
+    }
+  }
+}
+```
+
+In the example above, Vuex ORM will use `imageableType` from the `Image` model to determine the model to query and then
+match the `id`.
```

Comment:
`MorphTo` is not currently ready for use so we can either remove / comment this out or add additional documentation stating that it is not ready yet.

Reply:
Yeah we should simply remove this at the moment. Only document what's available at the moment make more sense in my opinion 👀